### ISSUE TITLE
Planetary logistics is improperly removing resources, cannot turn off warehouse in SPH/VAB

### ISSUE THREAD

**Issue by ss8913** [UmbraSpaceIndustries/USITools#44]
When launching with a 3.75m or 5.00m Kontainer, planetary logistics removes half of the resources on launch due to the Warehouse function, or at least this is the speculation.  There is no way to turn off the Warehouse function on a Kontainer from within SPH/VAB.  Having such a function in SPH/VAB would likely solve the problem, or at least it seems like it would.

**Comment by liTeWave1** [comment]
Sorry if this is not the right place to leave this comment, but new to GitHub. Think it is though. I'm experiencing a very similar issue, but under different circumstances. Have two ships with a 3.75m Kontainer landed on Minimus (mining ship/base and delivery ship). They are docked. The Kontainer of the mining base is full (32500/32500) of exotic minerals. The other container is about 2/3 full of exotic minerals. I transfer exotic minerals to the Kontainer from the mining base to the delivery ship, so that it is full. All of a sudden (no intervention on my part) both containers are exactly half full at 16250/32500 (some exotic minerals disappeared). I try to transfer again (I have tried both manually and with TAC Fuel balancer), transfer completes. A couple of seconds later, both Kointainers are exactly half full again. I have also tried to undock the ships right after transfer completes, does not help, next time I check both Kontainers are half full. Unlike previous report, my situation hence does not involve the SPH/VAB or launching.

I have tried to undock the craft, save and exit game, and then edit my persistence file to fill both containers (filled to 32000/32000). On load again contents of both ships next to each other are modified to half full.

Next, I tried launching the delivery ship into 50k orbit (so no longer next to mining base), saved game and exited. Again edited my persistence file to fill both containers (filled to 31499/32000). On load delivery ship in orbit now correctly has 31499/32000 exotic minerals. However, Kontainer in mining base still reverts to 16250/32500.

Incidentally, I am using UKS-LS and USI-MKS Lite.... MKS/OKS not installed.

Edit: Have researched further. I see what is happening is that the Kontainer is transferring half its contents to planetary logistics, which is expected behaviour for that system. The thing is I had no idea this feature/system even existed. It seems to me that it might perhaps be a feature intended for MKS which is spilling over into MKS Lite (and confusing the hell out of simple gits like me). Perhaps warehousing on Kontainers should be disabled on MKS Lite only installs?

Edit 2: I spoke too early. This IS an issue. Even **with warehousing turned OFF on both containers**: 1) an empty container landing on Minimus is automatically half filled from resources in planetary logistics, 2) the container on the mining base cannot fill up as anything over half full goes to planetary logistics and 3) it's impossible to transfer resources between the ships, they are both stuck on half full.
